src/PKSim.Infrastructure/Reporting/TEX/Reporters/ProjectReporter.cs

```diff
@@ -35,6 +35,7 @@ public override IReadOnlyCollection<object> Report(PKSimProject project, OSPSuit
          list.AddRange(_simulationsReporter.Report(project.All<Simulation>().ToList(), tracker));
          list.AddRange(_simulationComparisonsReporter.Report(project.AllSimulationComparisons.ToList(), tracker));
          list.AddRange(_observedDataReporter.Report(project.AllObservedData.ToList(), tracker));
+         list.AddRange(_observedDataReporter.Report(project.AllImporterConfigurations.ToList(), tracker));
```

Comment:
I don't think this makes sense to export this info to PDF. 
I would remove it for now (or you also need to create the Exporter for PDF for a configuration)

Reply:
Done